changelog/body.dd

```diff
@@ -0,0 +1,6 @@
+Generate header files using `do` instead of `body` as per DIP1003
+
+Support for 
```

Comment:
Mind the errors about trailing whitespace:

```
Searching for trailing whitespace
./changelog/2.078.0_pre.dd:33:Support for 
./changelog/2.078.0_pre.dd:34:[DIP1003](https://github.com/dlang/DIPs/blob/master/DIPs/DIP1003.md) 
./changelog/2.078.0_pre.dd:35:was added in release 2.075.0. Use of `body` in an error message and 
```

Reply:
Now fixed.